README.md

```diff
@@ -9,17 +9,17 @@ utility in their use.
 ElementsProject.org is a static website powered by [Hexo](https://hexo.io).
 You'll need `npm` (which comes with Node.js) to get started.
 
-First, install hexo: `npm install hexo-cli -g`
-
 Fork the repository (top right hand corner on GitHub), and get a local copy:
 ```
 git clone git@github.com:YOUR_USERNAME/elementsproject.org # clone the repo
 cd elementsproject.org # move into the directory
 ```
 
+Next, install the dependancies using `npm install`
```

Comment:
Yes. Since it uses `npm start` now instead of `hexo` directly, it'll automatically look into `./node_modules/.bin/` for the PATH lookup on the binary. Those CLI packages are basically just a proxy for exactly that. As a side note, i highly recommend adding path that entry to your bashrc.